### ISSUE TITLE
Unable to register user

### ISSUE THREAD

**Issue by baqar** [IBM-Blockchain-Archive/marbles#83]
I am getting the following error while trying to run marbles. I have the blockchain hosted over ibm bluemix. How can this issue be resolved ?

```
Running using Developer settings
- Tracking Deployment
loading hardcoded peers
loading hardcoded users
[ibc-js] Peer:  051ebddb7d8d4039b469f45524b698b6-vp2-051ebddb7d8d...:5004
[ibc-js] Registering  051ebddb7d8d4039b469f45524b698b6-vp2-051ebddb7d8d...:5004
 w/enrollId - user_type1_0
[ibc-js] Register - failure x1 : user_type1_0 401
[ibc-js]        going to try to register again in 30 secs
[ibc-js] Registering  051ebddb7d8d4039b469f45524b698b6-vp2-051ebddb7d8d...:5004
 w/enrollId - user_type1_0
[ibc-js] Register - failure x2 : user_type1_0 401
! looks like an error loading the chaincode or network, app will fail
 { name: 'register() error',
  code: 401,
  details: { Error: 'rpc error: code = 2 desc = Identity or token does not match
.' 
```

**Comment by dshuffma-ibm** [comment]
It doesn't like your enrollId / enrollSecret. Did you edit the creds file? [Setup](https://github.com/IBM-Blockchain/marbles/blob/v2.0/docs/use_local_hyperledger.md#finished)

**Comment by baqar** [comment]
I was using the credentials as generated by the server. However, I was able to resolve this with the following additional changes in it:

- The url for peers generated with the server credentials link in the block chain network used http, changed them over to https
- Also I had already registered this user to one of the nodes, since marbles app registers to the first node I updated peer ordering such it to registered to the correct node. 

One or both of these changes got the application working again for me.
